bookStore.css

```diff
@@ -7,23 +7,80 @@
 }
 
 body {
+  /* display: flex;
+  flex-direction: column;
+  justify-content: space-between; */
```

Comment:
- [ ] It looks like these comments doesn't have any necessary action in this project. Please, remove them if you do not need them. Thank you!